server/controller/questions.js

```diff
@@ -28,3 +28,28 @@ exports.post = async (request, response) => {
     response.status(500).send('Server Error');
   }
 };
+
+exports.put = async (request, response) => {
+  try {
+    const { id: loggedUserId } = request;
+    const data = request.body;
+    const { userId, question, questionId: id } = data;
+    if (loggedUserId === userId) {
+      const updateResult = await questions.update(
+        { question },
+        { where: { id } },
+      );
+      if (updateResult.length) {
+        response
+          .status(200)
+          .send('Your question has been successfully modified');
```

Comment:
@HemaSAli Here, instead of sending text, you could send the entire updated question object back.  This is a pattern that I see / use frequently with REST endpoints: any edit / update request will respond with the updated object.

Your frontend doesn't have to do anything with that data right now, but it's a good habit to get into.  For example, if your question object had a last_modified timestamp (that SQL automatically updates for you), then this is how you'd get that updated last_modified value back to your frontend.  Also, if there are other updates that your frontend isn't yet aware of, this PUT would update your frontend (if two people were editing different fields of the same object at the same time, this would send those other updates).

It's not an urgent change that you need to make, but something you could do the next time / for the next project :)